graphics.py

```diff
@@ -0,0 +1,90 @@
+import pygame
+from maze_generation import *
+from player import *
+
+WHITE = (255, 255, 255)
+GREY = (200, 200, 200)
+LIGHT_BLUE = (0, 244, 244)
+PINK = (252, 180, 213)
+BROWN = (220, 88, 5)
+YELLOW = (255, 255, 0)
+GOLD = (249, 166, 2)
+DUSTY = (207, 181, 59)
+
+def draw_maze(maze, surface):
+    cell_size = 30

```

Comment:
Should be equal everywhere in program so should be in other place (Ex. global variable or some class variable)